### ISSUE TITLE
[BottomSheet] Should have a dedicated dismiss button in VoiceOver

### ISSUE THREAD

**Issue by samnm** [material-components/material-components-ios#2267]

**Comment by ianegordon** [comment]
There is already a dedicated "Go Back / Dismiss" gesture in VO.  Make a Z with two fingers.  Do we need another method of dismissing a bottom sheet?

**Comment by ianegordon** [comment]
If the Z-gesture dismisses a bottom sheet, let's close this.
